### PR TITLE
 NMS-9673: Fixed XSS in userGroupView

### PR DESCRIPTION
**Pull request by fooker**
* JIRA: http://issues.opennms.org/browse/NMS-9645

This needs https://github.com/OpenNMS/opennms/pull/1717 in before.

### CODE REVIEW

**Comment by j-white** [comment]
@fooker can you rebase this against `foundation-2016`? https://github.com/OpenNMS/opennms/pull/1717 was merged but the commits still show up here.

**Comment by fooker** [comment]
Rebased

File: opennms-webapp/src/main/webapp/admin/userGroupView/roles/view.jsp [hunk 1]
```diff
@@ -148,14 +149,14 @@
 
     <tr>
       <th>Description</th>
-  	<td colspan="3">${role.description}</td>
+  	<td colspan="3"><%= request.getAttribute("role") == null ? "" : WebSecurityUtils.sanitizeString(((WebRole) request.getAttribute("role")).getDescription()) %></td>
```
Comment (j-white) [inline opennms-webapp/src/main/webapp/admin/userGroupView/roles/view.jsp]: Is there any reason why this one contains a null check, but the others on this page don't?

**Comment by RangerRick** [comment]
I imagine it's impossible to not have a role, but guarding against nulls doesn't hurt either.  One way or the other, can we merge this before Thursday? :)

**Comment by j-white** [comment]
I think the `WebSecurityUtils.sanitizeString` calls can be simplified by using `<c:out>` tags instead. If we don't get around to this before Thursday, I'd say we merge it as it though.

**Comment by j-white** [comment]
Closing in favor of https://github.com/OpenNMS/opennms/pull/1763, using tags makes this more manageable and avoid having to case the types.

**Pull request closed without merging**